documentation/src/main/markdown/packagemgrs/gradle.md

```diff
@@ -44,3 +44,7 @@ The init-detect.gradle script configures each project with the custom 'gatherDep
 The buildless gradle detector uses Project Inspector to find dependencies and does not support dependency exclusions.
 
 It currently supports "build.gradle" and does not support Kotlin build files.
+
+[company_name] [solution_name] 9.5.0 now uses the latest version of Project Inspector i.e. 2024.2.0 in which support certain arguments have been removed. 
```

Comment:
Suggested tweak to be a bit more generic in regard to versions and future releases. (Such that we won't need to update this with future Detect or PI release numbers etc.)
... and just to check, is the last parameter correct for the Gradle page? (`--conf "maven.repo:url"`)


As of [company_name] [solution_name] 9.5.0 the version of Project Inspector in use supports the `--build-system GRADLE` argument in place of `--strategy GRADLE`.
The `--force-gradle-repos "url"` argument will be removed from support in the next [company_name][solution_name] major release and replaced with the `--conf "maven.repo:url"` argument.

Reply:
Yes, the parameter is correct, this is mentioned in PI doc.